specification/azurestackhci/AzureStackHCI.StackHCIVM.Management/main.tsp

```diff
@@ -0,0 +1,48 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "./models.tsp";
+import "./GalleryImages.tsp";
+import "./LogicalNetworks.tsp";
+import "./MarketplaceGalleryImages.tsp";
+import "./NetworkInterfaces.tsp";
+import "./NetworkSecurityGroups.tsp";
+import "./SecurityRules.tsp";
+import "./StorageContainers.tsp";
+import "./VirtualHardDisks.tsp";
+import "./VirtualMachineInstance.tsp";
+import "./HybridIdentityMetadata.tsp";
+import "./AttestationStatus.tsp";
+import "./GuestAgent.tsp";
+import "../Operations.Management/models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.Versioning;
+/**
+ * Azure Stack HCI management service
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "HCI itself is a short form."
+@armProviderNamespace
+@service({
+  title: "Microsoft.AzureStackHCI",
+})
+@versioned(Versions)
+@armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5)
```

Comment:
This is a breaking change, I think you need to use v3, to avoid the break

Reply:
Discussed offline and agreed that this is ok to use